styles/addorder.css

```diff
@@ -0,0 +1,83 @@
+/* InputField.CSS */
+
+.inputField {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+
+  gap: 0.313rem;
+  margin-bottom: 0.8rem;
+}
+
+.inputField label {
+  line-height: 19px;
+}
+
+.inputField[validationState="0"] label {
+  color: var(--clr-white);
+}
+
+.inputField[validationState="1"] label {
+  color: var(--clr-primary);
+}
+
+.inputField[validationState="-1"] label {
+  color: var(--clr-error);
+}
+
+.inputField input,
+.inputField select {
+  background-color: hsla(0, 0%, 0%, 0);
+
+  color: var(--clr-white);
+  padding: 12px 16px;
+
+  box-sizing: border-box;
+  width: 600px;
+  height: 43px;
+  border-radius: 7px;
+}
+
+.inputField[validationState="0"] input {
+  border: 2px solid var(--clr-white);
+}
+
+.inputField[validationState="1"] input {
+  border: 2px solid var(--clr-primary);
+}
+
+.inputField[validationState="-1"] input {
+  border: 2px solid var(--clr-error);
+}
+
+.inputField::placeholder {
+  color: var(--clr-placeholder);
+}
+
+.inputField span {
+  font-size: 0.75rem;
+}
+
+.inputField[validationState="0"] span {
+  visibility: hidden;
+}
+
+.inputField[validationState="1"] span {
+  color: var(--clr-primary);
+}
+
+.inputField[validationState="-1"] span {
+  color: var(--clr-error);
+}
+
+/* Add Input Field */
+
+#row_2 {
+  display: none;
+}
+.addorder,
+.editmember {
+  padding-top: 4rem;
+  margin: auto;
+  width: fit-content;
+}
```

Comment:
Is the Member Search form relying on the addorder.css file?? Should it have it's own CSS file or is this fine?

Reply:
The styling also doesn't currently match Figma either.

Reply:
Yes, basically both pages need the same CSS code